server/src/routes/api/v1/pizzaPlaceReviewRouter.js

```diff
@@ -22,4 +22,12 @@ pizzaPlaceReviewRouter.post("/", async (req, res) => {
   }
 })
 
+pizzaPlaceReviewRouter.delete("/:id", async (req, res) => {
+  try {
+    await Review.query().deleteById(req.params.id)
+    return res.status(204).json({message: 'deletion success'})
+  } catch(error) {
+		return res.status(500).json({errors: error})
+	}
+})
```

Comment:
it could just be my view, but is the indentation off here?

Reply:
i promise it doesn't look like that on my machine